tests/test_sandbox_scripts.py

```diff
@@ -109,6 +109,10 @@ def test_sweep_reads():
 
     print os.listdir(in_dir)
 
+    assert os.path.exists(out1)
```

Comment:
On Wed, Mar 11, 2015 at 03:52:14PM -0700, Michael R. Crusoe wrote:
> > @@ -109,6 +109,10 @@ def test_sweep_reads():
> >  
> >      print os.listdir(in_dir)
> >  
> > +    assert os.path.exists(out1)
> 
> I can pull them out separately if you want.

Just curious why they're part of this PR, which promises to be rather
far reaching - yes, might be better to split off the cleanup changes.